sympy/physics/units/definitions.py

```diff
@@ -153,7 +153,7 @@
 # vacuum impedance:
 Z0 = vacuum_impedance = Quantity("vacuum_impedance", impedance, u0 * c)
 # Coulomb's constant:
-coulomb_constant = coulombs_constant = electric_force_constant = Quantity("coulomb_constant", force*length**2/charge**2, 1/(4*pi*vacuum_permittivity), abbrev="k_e")
+Ke=coulomb_constant = coulombs_constant = electric_force_constant = Quantity("coulomb_constant", force*length**2/charge**2, 1/(4*pi*vacuum_permittivity), abbrev="k_e")
```

Comment:
Space is still missing here.

I'm not a fan of seeing such huge line which make it really hard to see in editor. Can you please make lines <= 79 characters? Just for the `Ke` and `R`, no need to touch other units for now.